src/asm-manual.adoc

```diff
@@ -1079,6 +1079,37 @@ srli rd, rd, XLEN - 32
 |jr rs                        | jalr x0, rs, 0                                                | Jump register |
 |jalr rs                      | jalr x1, rs, 0                                                | Jump and link register |
 |ret                          | jalr x0, x1, 0                                                | Return from subroutine |
+|vfneg.v vd, vs               | vfsgnjn.vv vd, vs, vs                                         | Floating-point vector negate |
+|vfabs.v vd, vs               | vfsgnjx.vv vd, vs, vs                                         | Floating-point vector absolute value |
+|vmclr.m vd                   | vmxor.mm vd, vd, vd                                           | Vector clear mask register |
+|vmfge.vv vd, va, vb, vm      | vmfle.vv vd, vb, va, vm                                       | Vector Floating-point >=|
+|vmfgt.vv vd, va, vb, vm      | vmflt.vv vd, vb, va, vm                                       | Vector Floating-point >|
+|vmmv.m vd, vs                | vmand.mm vd, vs, vs                                           | Vector copy mask register |
+|vmnot.m vd, vs               | vmnand.mm vd, vs, vs                                          | Vector invert mask bits|
+|vmset.m vd                   | vmxnor.mm vd, vd, vd                                          | Vector set all mask bits|
+|vmsge.vi vd, va, i, vm       | vmsgt.vi vd, va, i-1, vm                                      | Vector >= Immediate|
+|vmsgeu.vi vd, va, i, vm      | vmsgtu.vi vd, va, i-1, vm                                     | Vector >= Immediate, unsigned|
+|vmsge.vv vd, va, vb, vm      | vmsle.vv vd, vb, va, vm                                       | Vector >= Vector|
+|vmsgeu.vv vd, va, vb, vm     | vmsleu.vv vd, vb, va, vm                                      | Vector >= Vector, unsigned |
+|vmsge.vx vd, va, x, vm       | vmsle.vx vd, x, va, vm                                        | Vector >= scalar|
```

Comment:
This is the not the correct expansion for vmsge.vx

From the spec.

```
  The RISC-V Instruction Set Manual Volume I | © RISC-V International
pseudoinstruction: vmsge{u}.vx vd, va, x, v0.t, vt
    expansion: vmslt{u}.vx vt, va, x;  vmandn.mm vt, v0, vt;  vmandn.mm vd, vd,
  v0;  vmor.mm vd, vt, vd
    The vt argument to the pseudoinstruction must name a temporary vector
  register that is
not same as vd and which will be clobbered by the pseudoinstruction
```

Reply:
@topperc  How can I define this different implementations and follow manual style? Shall I just do a * like in li?

Reply:
There are 3 different pseudoinstructions there that should each have their own row

vmsge{u}.vx vd, va, x
vmsge{u}.vx vd, va, x, v0.t // add a note in the last column that vd can't be v0
vmsge{u}.vx vd, va, x, v0.t, vt // you can write the 4 instruction expansion and a note that 2 instructions aren't needed if vd==v0.

Reply:
@topperc In the last commit I kept it exactly as described in the ISA Manual because I feel the 1:1 mapping is easier to understand, is this ok?

Reply:
Yes